public/stylesheets/main.css

```diff
@@ -92,6 +107,12 @@ nav span {
     transition: .5s ease-in-out;
 }
 
+#regBtn:hover #regBtn2:hover {
```

Comment:
Not really a change but if 2 ids have the same css...make it a class instead like .regBtn